configserver/src/main/java/com/yahoo/vespa/config/server/ConfigServerBootstrap.java

```diff
@@ -62,6 +65,7 @@ public void run() {
             log.log(LogLevel.INFO, "All applications redeployed");
         }
         versionState.saveNewVersion();
+        stateMonitor.status(StateMonitor.Status.up);
         log.log(LogLevel.DEBUG, "Starting RPC server");
         server.run();
         log.log(LogLevel.DEBUG, "RPC server stopped");
```

Comment:
It will only be reached for a short time when the server goes down (that's why I didn't set it), but thinking about it I think it's correct to set it to down anyway.